### DISCUSSION TITLE
How can I scroll back after #split ?

### DISCUSSION THREAD

**Discussion by Zhiming-Huang** [scandum/tintin#88] [General]
Hi Guys, I just have a quick question:
After I use #split to split a top bar, how can I view my past messages? If without a top bar, it is convenient just by using my mouse to scroll up to see the past messages. However, after splitting, the past messages are "eaten" by the top bar, and I can no longer access to them by using the mouse. So how can I do in this case?

**Comment by scandum** [comment]
Hello.

You can use page up and down to scroll.

You can also use `#config mouse on` and set `#event {SCROLLED MOUSE WHEEL UP} {#buffer up 1}`. Keep in mind you have to hold the shift key to copy/paste while using mouse mode.

**Reply by Zhiming-Huang** [reply]
Thank you very much for your suggestions!
